cypress/support/pageObjects/contestPage.ts

```diff
@@ -134,8 +134,8 @@ export class ContestPage {
 
   createContest(contestOptions: ContestOptions, users: Array<string>, shouldShowIntro: boolean = true): void {
     cy.createContest(contestOptions, shouldShowIntro);
-
     cy.location('href').should('include', contestOptions.contestAlias);
+    cy.find('a[data-contest-new_form]').trigger('click');
```

Comment:
The correct funcion should be `cy.get`:
```suggestion
    cy.get('a[data-contest-new-form]').trigger('click');
```

Reply:
This was the reason for cypress test failure :3
Thank you.